flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js

```diff
@@ -262,12 +262,13 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
             }, this.getDelaySwitchPage());
         };
 
-        CDUMenuPage.ShowPage(this);
+        //CDUMenuPage.ShowPage(this);
 
         this.onAirport = () => {
-            this.addNewMessage(NXFictionalMessages.notYetImplemented);
+            //this.addNewMessage(NXFictionalMessages.notYetImplemented);
+            this.onAirport();
         };
```

Comment:
I have a bit of a hard time understanding what this code does. Does this not just call itself recursively?

Reply:
It does indeed whilst also being outdated, that method is outdated and the pr needs a rebase.

Reply:
@derl30n can you add me as a friend on Discord?  Id like to DM or chat about this and below item to understand better what is going on. 
Thanks